test/test-helpers/create-entities.js

```diff
@@ -242,15 +261,14 @@ export async function createWork(optionalBBID) {
 	await new Entity({bbid, type: 'Work'})
 		.save(null, {method: 'insert'});
 	await createEntityPrerequisites(bbid);
-	// await createRelationshipAndRelationshipSet(bbid, uuidv4(), 'Author');
 	const languageSetId = await createLanguageSet();
 
 	const workAttribs = {
 		bbid,
 		languageSetId,
-		typeId: setData.id
+		typeId: random.number()
 	};
-	await new WorkType({...setData, label: 'Work Type 1'})
+	await new WorkType({id: workAttribs.typeId, label: `Work Type ${workAttribs.typeId}`})
 		.save(null, {method: 'insert'});
 	await new Work({...entityAttribs, ...workAttribs})
```

Comment:
```
return new Work({...entityAttribs, ...workAttribs})
		.save(null, {method: 'insert'}); 
```

Reply:
I don't think that's a bad idea at all, and could be useful i some cases.